src/main/java/racing/domain/CarMovableStrategy.java

```diff
@@ -4,17 +4,17 @@
 
 public class CarMovableStrategy implements MovableStrategy {
 
-    private final static int RANDOM_RANGE = 10;
-    private final static int RACING_CONDITION = 4;
-    private final static int RUN = 1;
-    private final static int STOP = 0;
+    private static final int RANDOM_RANGE = 10;
+    private static final int RACING_CONDITION = 4;
+    private static final boolean RUN = true;
+    private static final boolean STOP = false;
 
     private final Random RANDOM = new Random();
 
     @Override
-    public int randomMovable() {
+    public boolean randomMovable() {
         int random = RANDOM.nextInt(RANDOM_RANGE);
-        if(RACING_CONDITION <= random) {
+        if (RACING_CONDITION <= random) {
             return RUN;
         }
         return STOP;
```

Comment:
boolean 리턴으로 변경되었네요 👍 
조금 더 심플하게 코드를 변경하면 어떨까요?
```suggestion
        return RACING_CONDITION <= random;
```

Reply:
오 이런방법이 감사합니다 👍